src/libopensc/card-atrust-acos.c

```diff
@@ -116,7 +116,8 @@ static int atrust_acos_init(struct sc_card *card)
 
 	/* set the supported algorithm */
 
-	flags = SC_ALGORITHM_RSA_PAD_PKCS1 
+	flags = SC_ALGORITHM_RSA_PAD_PKCS1_TYPE_01
+		| SC_ALGORITHM_RSA_PAD_PKCS1_TYPE_02
```

Comment:
I would preserve the old flag as an alias for the two new ones to simplify the init functions of the drivers. In the init, the driver developer does not care if the PKCS1.5 padding is type 1 or 2 most of the time.